el-debate/src/main/java/pl/elpassion/elspace/debate/DebatesRepository.kt

```diff
@@ -5,25 +5,30 @@ import com.elpassion.android.commons.sharedpreferences.createSharedPrefs
 import com.elpassion.sharedpreferences.gsonadapter.gsonConverterAdapter
```

Comment:
fun areCredentialsAvailable(token: String): Boolean
and impl:
override fun areCredentialsAvailable(token: String): Boolean = credentialsRepository.read(token) != null